### ISSUE TITLE
Support Windows 10 UWP apps (uap10.0)

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#615]
Add a project that targets Windows 10 / Universal Windows apps

**Comment by objorke** [comment]
Or is this not needed? Can the Windows 8.1 package be used?

**Comment by csabar** [comment]
I have successfully compiled OxyPlot.Windows to support UAP, but it was not so straightforward for me. The binaries are not compatible, but the source code is compatible. I think the only change I made was adding the UNIVERSAL definition to a default UAP project. But you definitively need at least separate nuget package.

**Comment by objorke** [comment]
There is a nuget restore error on appveyor, anyone knows why? see #616

**Comment by objorke** [comment]
the UAP project is built and published to NuGet (included in the OxyPlot.Windows package), but it seems like the OxyPlot.Core package is not working with UAP

**Comment by tibel** [comment]
What do you mean with not working?

**Comment by objorke** [comment]
When I add the OxyPlot.Windows package to a UAP10.0 app, the OxyPlot.Core package is not included in the references. Shouldn't it be there? Or is this correct? It seems to build fine, but does not run. The example in the `OxyPlot.Windows.sln` is working... Will try to replace the project references with NuGet references...

**Comment by objorke** [comment]
I opened OxyPlot.Windows.sln and changed the Windows10/SimpleDemo project:
I replaced the OxyPlot and OxyPlot.Windows10 references with the latest OxyPlot.Windows NuGet package - it builds ok but does not run...

**Comment by tibel** [comment]
What I don't understand is, why win10 OxyPlot looks different in nuget than win81?
- native is not needed for a class library, only for apps; stay with AnyCPU
- nuspec for win10 looks strange with native parts

It should look like this:
``` xml
<file src="..\..\Output\Windows10\OxyPlot.Windows10.???" target="lib\uap10.0" />
<file src="..\..\Output\Windows10\Themes\*.*" target="lib\uap10.0\OxyPlot.Windows10\Themes" />
```
As simple as for portable-windows8+wpa81, no need for other stuff. Maybe that is your problem?

**Comment by objorke** [comment]
I could not get AnyCPU to build on AppVeyor. Can try again!
Thanks, will try changing the nuspec!

**Comment by GeertvanHorrik** [comment]
I was just trying this out, so this is very recent. Whenever I use this, I get the exception that "Oxyplot.Windows" cannot be loaded. Maybe because it's called OxyPlot.Windows10 for UA{?

**Comment by GeertvanHorrik** [comment]
A few changes that I did to make this work (locally):

1. Change assembly name

![image](https://cloud.githubusercontent.com/assets/1246444/12138480/91a632c8-b459-11e5-895c-b919f573ed36.png)

2. Update configuration

![image](https://cloud.githubusercontent.com/assets/1246444/12138497/b507cd9e-b459-11e5-8352-8cdfae2ab0fe.png)

Now I can use OxyPlot in my UAP app.

**Comment by objorke** [comment]
@tibel and @GeertvanHorrik thanks for the comments!
The nuget package OxyPlot.Windows 1.0.0-unstable1966 now works for Univeral Windows (v10.0) apps on my machine! :-)
I do not understand why AnyCPU did not work in the AppVeyor build of my previous pull request.

**Comment by objorke** [comment]
It seems like the platform must be specified as a build_script in the appveyor.yml
See #726 that fails

**Comment by HammadRehman** [comment]
Greetan I would like to ask are you developing "Universal windows app" using C++/Xaml or C#/XAML project?
because I want to use OxyPlot in UWAP C++/Xaml project. I am able to use it in C# based project but not in C++/Xaml project. First I tried to add it using Nuget package manager but it didn't install. Then I followed this (https://social.msdn.microsoft.com/Forums/en-US/8b77c775-21e0-4b43-b9f3-fb3777c43212/adding-libraries-and-use-them?forum=winappswithnativecode) (jrboddie's answer). In this Windows runtime component (with OxyPlot) is created in C# and used in C++ project. Here I am able to build the project but app crashes saying OxyPlot (added in C# project) not found.
